index.html

```diff
@@ -81,7 +81,7 @@ <h3>LET'S CONNECT</h3>
             </div>
 
             <button type="button" class="aboutbutton">
-                    Get my resumeg
+                    Get my resume
                 </button>
 
```

Comment:
- [ ] Following the project requirements, You should build a contact form in a footer for this milestone. Kindly add a contact form with these [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).